backend/api/memory.py

```diff
@@ -49,4 +49,29 @@ def get_memory_by_id(memory_id):
     finally:
         if conn:
             conn.close()
-    return memory
\ No newline at end of file
+    return memory
+
+
+def get_latest_memory():
```

Comment:
@Caleb-Rogers  let me know if this is problematic

Reply:
My PR has this same method name which will cause a merge conflict